src/utilities/dnd/DndWrapper.tsx

```diff
@@ -0,0 +1,24 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { PropsWithChildren, memo, useEffect, useState } from 'react';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+
+interface DndWrapperProps {
+    id: string;
+}
+
+export const DndWrapper = memo(({ id, children }: PropsWithChildren<DndWrapperProps>) => {
```

Comment:
I would avoid using PropsWithChildren as children should be mandatory in this case.